bugbear.py

```diff
@@ -1530,6 +1527,12 @@ def visit_Lambda(self, node):
         " greater to provide more information to the user."
     )
 )
+B029 = Error(
+    message=(
+        "B029 Using `except` with an empty tuple does not handle anything. "
+        "Add exceptions to handle instead."
```

Comment:
```suggestion
        "B029 Using `except: ()` with an empty tuple does not handle/catch "
        "anything. Add exceptions to handle."
```

Yup - that is also a '0' !